src/resources/raids/policestation.ts

```diff
@@ -150,8 +150,8 @@ export const station: Location = {
 		},
 		{
 			type: 'LootChannel',
-			name: 'dereks-hunting-shop',
-			display: 'Dereks Hunting Shop',
+			name: 'Weapons and Ammunition Room',
```

Comment:
names must follow discord channel naming rules (lowercase and use - instead of spaces)
```suggestion
			name: 'weapon-room',
```